core/src/test/scala/unit/kafka/admin/DeleteTopicTest.scala

```diff
@@ -67,21 +66,21 @@ class DeleteTopicTest extends ZooKeeperTestHarness {
       servers.filter(s => s.config.brokerId != follower.config.brokerId)
         .forall(_.getLogManager().getLog(topicPartition).isEmpty), "Replicas 0,1 have not deleted log.")
     // ensure topic deletion is halted
-    TestUtils.waitUntilTrue(() => zkUtils.isTopicMarkedForDeletion(topic),
+    TestUtils.waitUntilTrue(() => zkClient.isTopicMarkedForDeletion(topic),
       "Admin path /admin/delete_topic/test path deleted even when a follower replica is down")
     // restart follower replica
     follower.startup()
-    TestUtils.verifyTopicDeletion(zkUtils, topic, 1, servers)
+    TestUtils.verifyTopicDeletion(zkClient, topic, 1, servers)
   }
 
   @Test
   def testResumeDeleteTopicOnControllerFailover() {
     val topicPartition = new TopicPartition("test", 0)
     val topic = topicPartition.topic
     servers = createTestTopicAndCluster(topic)
-    val controllerId = zkUtils.getController()
+    val controllerId = zkClient.getControllerId.getOrElse(throw new KafkaException("Controller doesn't exist"))
```

Comment:
Nit: seems like `TestUtils.fail` might be more appropriate than `KafkaException`. A few other similar cases.